flowforge/input/System.py

```diff
@@ -146,6 +149,8 @@ def __init__(
         self._gas = None
 
         self._BoundaryConditions = None  # ** Boundary Conditions **
```

Comment:
```suggestion
        self._BoundaryConditions = None
```
I also think the default should be a list or something? Easier for interacting if the list is empty? But None could raise errors - idk, just think about it.

Reply:
I'm down to switch this - I've done so in my latests commit. since these will return objects, I just created empty versions of these object for initialization